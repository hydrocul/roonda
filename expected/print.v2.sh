#!/bin/sh

#################################################
# roonda_58d917154968e6bb2c7cf57933ff1f8a3ab37183.sh:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_58d917154968e6bb2c7cf57933ff1f8a3ab37183.sh
echo sh
echo -n 123
echo -n '
'
echo abc
echo 'def
'
END_OF_ROONDA_SOURCE_FILE
#################################################
# roonda_a21c8c525b371d0504d4c11bec84150a69e43ab2.pl:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_a21c8c525b371d0504d4c11bec84150a69e43ab2.pl
use Encode qw/encode/;
use Data::Dumper;

print encode('utf-8', "perl" . "\n");
print encode('utf-8', 123);
print encode('utf-8', "\n");
print encode('utf-8', "abc" . "\n");
print Dumper("def\n");
END_OF_ROONDA_SOURCE_FILE
#################################################
# roonda_717b078fd14c465dd07cd51d7259e3870ed02c77.rb:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_717b078fd14c465dd07cd51d7259e3870ed02c77.rb
puts "ruby"
print 123
print "\n"
puts "abc"
p "def\n"
END_OF_ROONDA_SOURCE_FILE
#################################################
# roonda_4c12e0e6b2758d47a3c2355d0bb68fb5019280b8.py:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_4c12e0e6b2758d47a3c2355d0bb68fb5019280b8.py
import sys

print(str("python2"))
sys.stdout.write(str(123))
sys.stdout.write(str("\n"))
print(str("abc"))
print("def\n")
END_OF_ROONDA_SOURCE_FILE
#################################################
# roonda_f9375dae8afb70dc30187aa828ac96867c9bca7d.py:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_f9375dae8afb70dc30187aa828ac96867c9bca7d.py
import sys

print(str("python3"))
sys.stdout.write(str(123))
sys.stdout.write(str("\n"))
print(str("abc"))
print("def\n")
END_OF_ROONDA_SOURCE_FILE
#################################################
# roonda_e906d352ab6f549651b51c2a2cbb907b96213a86.php:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_e906d352ab6f549651b51c2a2cbb907b96213a86.php
<?php

echo "php" . "\n";
echo 123;
echo "\n";
echo "abc" . "\n";
var_export("def\n");
END_OF_ROONDA_SOURCE_FILE
#################################################

sh $ROONDA_TMP_PATH/roonda_58d917154968e6bb2c7cf57933ff1f8a3ab37183.sh
perl $ROONDA_TMP_PATH/roonda_a21c8c525b371d0504d4c11bec84150a69e43ab2.pl
ruby $ROONDA_TMP_PATH/roonda_717b078fd14c465dd07cd51d7259e3870ed02c77.rb
python2 $ROONDA_TMP_PATH/roonda_4c12e0e6b2758d47a3c2355d0bb68fb5019280b8.py
python3 $ROONDA_TMP_PATH/roonda_f9375dae8afb70dc30187aa828ac96867c9bca7d.py
php $ROONDA_TMP_PATH/roonda_e906d352ab6f549651b51c2a2cbb907b96213a86.php
