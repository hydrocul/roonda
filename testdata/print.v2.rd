sh v2

(sh
 (println "sh")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
(perl
 (println "perl")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
(ruby
 (println "ruby")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
(python2
 (println "python2")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
(python3
 (println "python3")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
(php
 (println "php")
 (print 123)
 (print "\n")
 (println "abc")
 (dump "def\n"))
