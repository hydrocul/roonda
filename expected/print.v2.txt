sh
123
abc
def

perl
123
abc
$VAR1 = 'def
';
ruby
123
abc
"def\n"
python2
123
abc
def

python3
123
abc
def

php
123
abc
'def
'